src/CSET/recipes/derived_diagnostics/daily_weather/daily_09_minimum_temperature_spatial_plot.yaml

```
@@ -0,0 +1,44 @@
+category: Daily Weather
+title: $MODEL_NAME Daily minimum temperature at 0900 UTC
+description: |
+  Extracts and plots the daily minimum temperature between 0900 UTC and 0900 UTC
+  the following day in $MODEL_NAME as per observation measurements. The minimum
+  is created from a rolling 24 h window, adds hour of day coordinate, and then
+  selects the 21 UTC hour from the rolling window as this represents the validity
+  time of the required range. The maximum temperature can occur any time in the 24
```

This is a minimum temperature diagnostic, should this be talking about Maximum temperature?

```suggestion
  time of the required range. The minimum temperature can occur any time in the 24
```

---

Thanks for spotting this, it should be talking about Minimum temperature, it was a typo.